project3/style.css

```
@@ -0,0 +1,60 @@
+body {
+    margin: 0;
+    background-color: rgb(243, 240, 240);
+}
+
+h1 {
+    padding: 20px;
+    background-color: rgb(128, 242, 250);
+    margin-top: 0;
+    margin-bottom: 0;
+    text-align: center;
+}
+
+main,
+aside,
+footer {
+    padding: 20px;
+    background-color: rgb(0, 217, 255);
+    margin-top: 0;
+    margin-bottom: 0;
+    text-align: left;
+}
+
+aside {
+    background-color: rgb(19, 186, 216);
+}
+
+footer {
+    background-color: rgb(41, 155, 207);
+}
+
+
+body {
+    color: black;
+    font-family: helvetica, Impact, 'Arial Narrow Bold', sans-serif;
+    font-size: 13px;
+    margin: 0;
+    text-align: left;
+}
+
+img {
+    size
+}
+
+svg.makeup {
+    margin-top: 3%;
+    margin-bottom: 0;
+    width: 5%;
+    height: 5%;
+}
+
+svg.research {
+    width: 5%;
+    height: 5%;
+}
+
+svg.food {
+    width: 5%;
+    height 5%;
```

missing the colon on this one:
```suggestion
    height: 5%;
```